src/containers/Exhibitions/index.jsx

```diff
@@ -1,18 +1,20 @@
 import { useState, useEffect } from 'react';
-import axios from 'axios';
 
+import Pagination from '@mui/material/Pagination';
+import Box from '@mui/material/Box';
 import ExhibitionCard from '../../components/ExhibitionCard';
```

Comment:
Same, please leave an empty line.

Reply:
@JuanVimon3 please take a closer look at this -- the ExhibitionCard component should be imported after leaving an empty line under the Box component. I didn't mean to add the empty line between ExhibitionCard and axiosInstance. So it should look like this:

```
import Pagination from '@mui/material/Pagination';
import Box from '@mui/material/Box';

import ExhibitionCard from '../../components/ExhibitionCard';
import axiosInstance from '../../axiosInstance';

import styles from './styles.module.css';
```